src/services/saveImagesToUSB.ts

```diff
@@ -0,0 +1,54 @@
+import { IService } from '../types';
+import { DiskUsage } from 'types/motionModel';
+import { readdirSync, existsSync, mkdirSync, rename } from 'fs';
+import { USB_WRITE_PATH } from 'config';
+import { getDateFromUnicodeTimestamp } from '../util';
+
+const diskUsage: DiskUsage = {};
+
+export const getDiskUsage = () => {
+    return diskUsage;
+};
```

Comment:
@HariHive this is a leftover from another service, right?

Reply:
yes right, removed